Trees Excercises.py

```diff
@@ -0,0 +1,77 @@
+#Trees Excercises
+
+#Tree Construction
+class TreeNode: 
+    def __init__(self, left, right, value):
+        self.left = left
+        self.right = right
+        self.value = value
+
+class Tree:
+    def __init__ (self, root):
+        self.root = root
+
+    def add_node (self, value):
+        node = TreeNode(
+            left = None,
+            right= None,
+            value = value
+        )
+        n = self.root 
+
+        def add_(node, n):
+ 
+            if node.value > n.value:
+                if n.right != None:
+                    add_(node, n.right)
+                else: 
+                    n.right = node 
+                    
+            elif node.value < n.value: 
+                if n.left != None:
+                    add_(node, n.left)
+                else: 
+                    n.left = node
+
+        add_(node, n)
```

Comment:
Generally for inner functions, you want to have the _ before the name, so something like

```
def _add():
    pass
```

Reply:
Cool, will do.